internal/graph/track_check_resolver.go

```diff
@@ -212,6 +212,7 @@ func (t *TrackerCheckResolver) ResolveCheck(
 		RequestMetadata:      req.GetRequestMetadata(),
 		VisitedPaths:         req.VisitedPaths,
 		Context:              req.GetContext(),
+		Consistency:          req.GetConsistency(),
```

Comment:
Can we replace with `resp, err := t.delegate.ResolveCheck(ctx, req)` ? I don't see a reason to create a brand new object that is the same as the input

Reply:
Makes sense to me, I wasn't sure if there was some reason for constructing a new object but looking at the associated PR it just seems like it was done that way from the start.

Done in f6ae94832a7690f07ea2d64e72f047a9b0f3c82c

Reply:
Yeah it's definitely not intuitive from a developer perspective when you're supposed to use the same one or clone it 😓 we need to improve this